submissions/Zakkarat/index.js

```diff
@@ -0,0 +1,92 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+const INHABITANT = {
+    species: {
+        human: 'human'
+    },
+    genders: {
+        male: 'male',
+        female: 'female'
+    }
+};
```

Comment:
I don't understand why do you need this constant object, and why you have only one specie here...

Reply:
This block is for things, which are appearing more than once. Actually I've done it because of the review on my previous [OOP exercise](https://github.com/kottans/frontend-2019-p2p/pull/25)

Reply:
So I think, you should move it. we've talked on meeting about this a little.

Reply:
So I should move it away and add every specie in classes?